includes/rest/class-followers.php

```diff
@@ -51,6 +51,10 @@ public static function register_routes() {
 	 * @return WP_REST_Response
 	 */
 	public static function get( $request ) {
+		$context = $request->get_param( 'context' );
+		if ( 'view' === $context ) {
```

Comment:
I would change the output of the `orderedItems` to have the ActivityPub headers and to not have too much duplicated content:

```PHP
$json->orderedItems = array_map(
	function( $item ) {
		if ( 'view' === $context ) {
			return $item->to_array();
		} else {
			return $item->get_url();
		}
	},
	Follower_Collection::get_followers( $user->get__id(), 20, $page )
);
```

Reply:
Hmm let's see if I can make that work, I like the idea of it